app/scripts/platforms/extension.js

```diff
@@ -60,6 +60,18 @@ class ExtensionPlatform {
     })
   }
 
+  updateWindowPosition (left, top, windowId) {
```

Comment:
Could we have the ID as the first arg here?

```suggestion
  updateWindowPosition (windowId, left, top) {
```